### PR TITLE
Fix token fiat prices for Gnosis tokens

### PR DESCRIPTION
**Pull request by DannyDelott**
Fiat prices for gnosis weren't resolving since the defillama id needed to be lower-case.

### CODE REVIEW

File: apps/hyperdrive-trading/src/ui/token/hooks/useTokenFiatPrice.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { parseFixed } from "@delvtech/fixed-point-wasm";
+import { useQuery } from "@tanstack/react-query";
+import { makeQueryKey } from "src/base/makeQueryKey";
+import { isTestnetChain } from "src/chains/isTestnetChain";
+import { Address } from "viem";
+import { useChains } from "wagmi";
+
+export function useTokenFiatPrice({
+  tokenAddress,
+  chainId,
```
Comment (cashd) [inline apps/hyperdrive-trading/src/ui/token/hooks/useTokenFiatPrice.ts:10]: is there ever a situation in which the chainId is not the same as `useChainId()`?

File: apps/hyperdrive-trading/src/ui/token/hooks/useTokenFiatPrice.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}: {
+  tokenAddress: Address | undefined;
+  chainId: number;
+}): {
+  fiatPrice: bigint | undefined;
+} {
+  const chains = useChains();
```
Comment (cashd) [inline apps/hyperdrive-trading/src/ui/token/hooks/useTokenFiatPrice.ts:17]: I started to use this as a replacement recently, a bit less mental steps. 

`useClient().chain.name`

Oh I wish `useChain()` existed

**Review by cashd — APPROVED** [review]
nice!

File: apps/hyperdrive-trading/src/ui/token/hooks/useTokenFiatPrice.ts (hunk 1 above)
Comment (DannyDelott) [reply apps/hyperdrive-trading/src/ui/token/hooks/useTokenFiatPrice.ts:10]: Yep, our components are not tied to the chain your wallet is connected to. You can be connected to mainnet but viewing a gnosis pool.

**Pull request merged**